src/Meshes/mesh.ts

```diff
@@ -2587,7 +2587,7 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
             }
 
             vertex_data.indices = indices;
-            vertex_data.applyToMesh(this);
+            vertex_data.applyToMesh(this, this.isVertexBufferUpdatable(VertexBuffer.PositionKind));

```

Comment:
Is there any better option to get information about mesh `updatable` option?

Reply:
Nope this is perfect!